scss/_patterns_buttons.scss

```
@@ -17,10 +16,8 @@ $hover-background-opacity-percentage: $hover-background-opacity-amount * 100%;
 }
 
 @mixin vf-button-white-success-icon {
-  %vf-button-white-success-icon {
-    & .p-icon--success {
-      @include vf-icon-success($color-x-light, $color-transparent);
-    }
+  @include deprecate('3.0.0', 'vf-button-white-success-icon mixin will be removed, use @extend %vf-button-white-success-icon instead') {
```

Just a rewording suggestion
```suggestion
  @include deprecate('3.0.0', 'vf-button-white-success-icon mixin will be removed as it is not needed, %vf-button-white-success-icon is part of base styles') {
```